airbyte-cdk/python/airbyte_cdk/sources/streams/http/http_client.py

```diff
@@ -137,20 +137,18 @@ def clear_cache(self) -> None:
         if isinstance(self._session, requests_cache.CachedSession):
             self._session.cache.clear()  # type: ignore # cache.clear is not typed
 
-    def _dedupe_query_params(self, url: str, params: Optional[Mapping[str, str]]) -> Mapping[str, str]:
+    def _dedupe_query_params(self, url: str, params: Optional[Mapping[str, str]] = None) -> Mapping[str, str]:
```

Comment:
why is this adding a default value?

Reply:
codeflash added it, possibly because the type annotation says `Optional` but then it isn't really optional unless it has a default value